web/models/user.ex

```diff
@@ -28,6 +28,9 @@ defmodule CodeCorps.User do
     field :username, :string
     field :website, :string
 
+    field :state, :string
```

Comment:
Oh I see that was there in the migration. Idk if you put that in `field`. My bad.